sdk/core/switcher/entry.S

```diff
@@ -97,6 +97,25 @@ switcher_scheduler_entry_csp:
 	bnez               t2, .Lforce_unwind
 .endm
 
+.macro zero_stack base top scratch
```

Comment:
Add a comment saying that `base` has its address changed and `scratch` is clobbered (obvs).

Reply:
Added.

Reply:
You forgot to mention that `top` is also modified.

Reply:
While we are at it you should mention the alignment requirements on base and top.